### PR TITLE
display access with icon instead of text #497

### PR DESCRIPTION
**Pull request by dartcafe**
display access with icon instead of text #497
bildschirmfoto vom 2019-02-05 um 06 52 58

### CODE REVIEW

**Comment by dartcafe** [comment]
Changed list order and mobile view.
bildschirmfoto vom 2019-02-05 um 07 03 55
bildschirmfoto vom 2019-02-05 um 07 04 41

**Comment by v1r0x** [comment]
Nice job! :+1: 

I'd resize the icons from `32px` to `16px`. They look really really big :wink: 
Same for the poll type icons.

polls-access-icon

File: src/js/components/pollListItem.vue [hunk 1]
```diff
@@ -299,4 +333,58 @@ export default {
 	background-color: var(--color-success);
 	border-radius: 50%;
 }
+
+
+.thumbnail.accessasd {
```
Comment (v1r0x) [inline src/js/components/pollListItem.vue]: is the "asd" at the end on purpose? :grin:

File: src/js/components/pollListItem.vue [hunk 2]
```diff
@@ -299,4 +333,58 @@ export default {
...
+    &.registered {
+		mask-image: var(--icon-group-000) no-repeat 50% 50%;
+        -webkit-mask: var(--icon-group-000) no-repeat 50% 50%;
+        mask-size: 32px;
+    }
+
+}
+
+
+.accessaa {
```
Comment (v1r0x) [inline src/js/components/pollListItem.vue]: Same for "aa"

**Comment by bpcurse** [comment]
> Nice job! +1
> 
> I'd resize the icons from `32px` to `16px`. They look really really big wink
> Same for the poll type icons.
> 
> polls-access-icon

The 32px version is large but it is in line with the user avatars and in my opinion seems to be adequate and looks smoother (just my personal opinion). Whereas the 16px version creates saw blade-like size changes.

Sorry to interfere with you developers. I hope you don't mind if I add my two cents here.

File: src/js/components/pollListItem.vue (hunk 1 above)
Comment (dartcafe) [reply src/js/components/pollListItem.vue]: Hehe. Was left from playing with the css attributes.

**Comment by dartcafe** [comment]
I agree with the access icons, but the icons in the first column look a little bit lost on 16px... 

grafik

**Comment by v1r0x** [comment]
@dartcafe Except the large green checkmark icons: I like it! The icons should help users not distract them from other important stuff (most important title). I'd keep it that way, but that's just my opinion :)

> Sorry to interfere with you developers. I hope you don't mind if I add my two cents here.

No problem, any additional point of view is more than welcome! :)

**Comment by dartcafe** [comment]
@v1r0x. I would prefer to leave them at 32px, because the indicator becomes to huge in comparism. 

I would keep the 32px solution for now. When switching to the navigation, these have to be resized at least to a lower size. 

With 32px
grafik

With 16 px
grafik

resizing the indicator as well
grafik

**Comment by v1r0x** [comment]
> resizing the indicator as well
> grafik

I like this version the most :)

**Comment by dartcafe** [comment]
Let us leave this for the next iterations. The small icons look a little bit lost for me.

**Comment by dartcafe** [comment]
I opened a new issue (#516) for discussion about the icon size, so I can merge this for now.

**Pull request merged**